src/design-tokens/tier-2-usage/colors-background.json

```diff
@@ -35,7 +35,12 @@
                 "value": "{eds.color.brand.grape.100}"
               },
               "strong": {
-                "value": "{eds.color.brand.grape.600}"
+                "@": {
+                  "value": "{eds.color.brand.grape.600}"
+                },
+                "hover": {
+                  "value": "{eds.color.brand.grape.800}"
+                }
```

Comment:
Added after discussion in [figma](https://www.figma.com/file/yLYL19XZI0SWhbQJYuKVso?node-id=855:14061#187355723)